packages/queued-request-controller/CHANGELOG.md

```diff
@@ -6,6 +6,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.1.3]
+### Changed
+- Bump dependency on @metamask/json-rpc-engine to ^7.2.0 ([#1895](https://github.com/MetaMask/core/pull/1895))
+- Bump @metamask/utils from 8.1.0 to 8.2.0 ([#1957](https://github.com/MetaMask/core/pull/1957))
+
+### Fixed
+- Fix network nicknames ([#2000](https://github.com/MetaMask/core/pull/2000))
```

Comment:
```suggestion
- Fix issue in extension with useRequestQueue toggled on, when a dapps selected network is different than the globally selected network, and when the dapps selected network is not a built in network, nickname would not appear in the 'toNetworkConfiguration' parameter passed to `addApproval` ([#2000](https://github.com/MetaMask/core/pull/2000))
```

Hows that?

Reply:
```suggestion
- Fixes an issue in the extension when 'useRequestQueue' is enabled. The problem occurred when a DApp's selected network differed from the globally selected network, and when the DApp's chosen network was not a built-in network. Under these conditions, the nickname would not be displayed in the 'toNetworkConfiguration' parameter passed to the `addApproval` function ([#2000](https://github.com/MetaMask/core/pull/2000)).
```
...Or made 'better' by gpt

Reply:
I like how it put the period after the parentheses lol.

But yes! That's very understandable!

Reply:
Either looks good to me!